base_layer/core/src/chain_storage/test/chain_storage.rs

```diff
@@ -646,10 +646,10 @@ fn restore_mmr() {
         .fetch_mmr_base_leaf_nodes(MmrTree::Header, 0, 100)
         .unwrap()
         .total_leaf_count;
-    assert_eq!(utxo_mmr_leaf_count, 11);
-    assert_eq!(kernel_mmr_leaf_count, 5);
-    assert_eq!(rp_mmr_leaf_count, 11);
-    assert_eq!(header_mmr_leaf_count, 4);
+    assert_eq!(utxo_mmr_leaf_count, 14);
+    assert_eq!(kernel_mmr_leaf_count, 8); // Todo This might be wrong, verify 8 + 2 +2 =12
```

Comment:
```suggestion
    assert_eq!(kernel_mmr_leaf_count, 8); 
```
Only 4 block in base mmr. 4(cb of gb,b1,b2 and b3)+4(tx1,tx2,tx3 and tx4)=8